JAMES-1765 Add a post about James 3.0 release

chibenwa opened the pull request:
Tweet : 

```
The Apache James team is delighted to finally announce the 3.0 version of the James server.

Thanks to all contributors!

[tiny url link to the post]
```

Here is the visual of the post:

![capture d ecran de 2017-06-19 10-11-56](https://user-images.githubusercontent.com/6928740/27268269-c516ae0a-54d8-11e7-87b4-3ea8261317a9.png)

quynhn commented on src/homepage/_posts/2017-06-20-james-3.0.markdown:
```diff
@@ -0,0 +1,36 @@
+of the **3.0 version** of the Apache James server.
+
+This release brings:
+ - **new protocols**: IMAP, [JMAP](https://jmap.io/) (experimental), [WebAdmin](https://james.apache.org/server/manage-webadmin.html).
```
IMAP is the new one on V3?

chibenwa replied:
Yes

aduprat replied:
Is WebAdmin a new protocol ?

mbaechler commented:
We should not say Guice is a packaging thing, it's a wiring. We should probably fix the doc too.
Guice wiring uses jar packaging + manifest.

rouazana commented on src/homepage/_posts/2017-06-20-james-3.0.markdown:
```diff
@@ -0,0 +1,36 @@
+ - **new protocols**: IMAP, [JMAP](https://jmap.io/) (experimental), [WebAdmin](https://james.apache.org/server/manage-webadmin.html).
+ - **new back-ends**: [Cassandra](https://cassandra.apache.org/) mailbox, [ElasticSearch](https://www.elastic.co/fr/products/elasticsearch)
+ search index, [Lucene](https://lucene.apache.org/) search index amongst others.
+ - **new [packaging](https://james.apache.org/server/packaging.html)**: Especially [Guice](https://github.com/google/guice)
```
Upcase for the 3 new, to be consistent with the following points

rouazana commented on src/homepage/_posts/2017-06-20-james-3.0.markdown:
```diff
@@ -0,0 +1,36 @@
+ community and being more attractive to newcomers.
+
+Special care have been took with testing. The team works in a Test Driven Development fashion, and improved the test coverage of the project.
+We write many integration tests with [MPT](https://james.apache.org/mpt/index.html), but also using HTTP, IMAP and SMTP libraries. We
```
s/write/wrote/ ?

rouazana commented on src/homepage/_posts/2017-06-20-james-3.0.markdown:
```diff
@@ -0,0 +1,36 @@
+your applications. Read [more](https://james.apache.org/index.html#intro).
+
+If you encounter any problem with this release, please contact us [on gitter](https://gitter.im/apache/james-project) and open tickets
+on [our bug tracker](https://issues.apache.org/jira/browse/JAMES).
```
what about mentionning the good old mailing lists?

chibenwa commented:
See for merging s/packaging/wiring/g fast : https://github.com/linagora/james-project/pull/851

strokyl commented on src/homepage/_posts/2017-06-20-james-3.0.markdown:
```diff
@@ -0,0 +1,37 @@
+of the **3.0 version** of the Apache James server.
+
+This release brings:
+ - **New protocols**: IMAP, [JMAP](https://jmap.io/) (experimental), .
```
we are not sure if only JMAP is experimental or both are.
I would separate IMAP and JMAP by a new line

chibenwa replied:
Other documents states it better like the feature support matrix with version. I would not sacrifice readability for it.

chibenwa replied:
I added a (stable) after IMAP

strokyl commented on src/homepage/_posts/2017-06-20-james-3.0.markdown:
```diff
@@ -0,0 +1,37 @@
+ [Kibana](https://www.elastic.co/fr/products/kibana) log collection...
+ - **Community enhancements**: New [welcoming website](https://james.apache.org/index.html), as well as documentation re-work.
+ Also we introduced a [community chat](https://gitter.im/apache/james-project). All of this aims
+     at getting a more living
```
You can merge those two line in one

strokyl commented on src/homepage/_posts/2017-06-20-james-3.0.markdown:
```diff
@@ -0,0 +1,37 @@
+your applications. Read [more](https://james.apache.org/index.html#intro).
+
+If you encounter any problem with this release, please contact us [on gitter](https://gitter.im/apache/james-project), on the
+ [mailing lists](https://james.apache.org/mail.html) and open tickets on [our bug tracker](https://issues.apache.org/jira/browse/JAMES).
```
or open tickets on

chibenwa replied:
Why ? Why can't I do them all?

strokyl replied:
or is not exclusive but and let think you have to do all of them

chibenwa replied:
Or implies imo that you have to do only one.
And it is better if the community uses all of these media, no?

strokyl replied:
I don't agree but I think there is not a universal answer to this question so let it like that

chibenwa commented:
Email (links might change) for Apache internal communication team:

```
Hello,

The Apache James project [1] is pleased to announce the release of
version 3.0.0 of the Apache James Server, a mail customisable mail
server handling common email protocols.

The version 3.0.0 brings new features like the IMAP protocol, and
additional back-ends for mail storage like a Cassandra based mailbox.
We also introduced the use of the Guice wiring, enabling choosing which components
to embed by code. An HTTP interface for administration was also added.

Apache James Server can be downloaded from the following page [2].
Binary artifacts are published on Maven Central under Group Id
"org.apache.james" . Read more details on this release in the Apache 
James news for release 3.0.0 [3].

Please, remember to check the signature and fingerprint of the
downloaded files.

Have a great day,
Benoit Tellier, on behalf of tha Apache James community.

[1] james.apache.org
[2] apache.org/dist/james/server/3.0.0
[3] link to be added when publishing release
```

chibenwa commented:
Team to James user ML (please review)

```
Hello,

Apache James developers are very glad to finally announce the release of Apache James Server 3.0.

After several years of efforts, it brings IMAP supports, JMAP support, additional back-end (mailbox on top of Cassandra, searching emails in ElasticSearch or Lucene), new wiring with Guice, an HTTP administration interface, a new website, and much more!

You can get more information on this release here: [link to the article].

You can download the release from here: [link to dowload].

Best regards,

[Sender]
Apache James PMC
```

aduprat commented:
:+1: for all

chibenwa commented:
Communication for V3 have already been done.
